Tests/PSGetFindScript.Tests.ps1

```diff
@@ -51,6 +52,130 @@ function SuiteCleanup {
     $null = Import-PackageProvider -Name PowerShellGet -Force
 }
 
+
+
+Describe FindScriptTests -Tags 'TDD' {
+
+    BeforeAll {
+        SuiteSetup
+    }
+
+    AfterAll {
+        SuiteCleanup
+    }
+
+    # Downlevel Tests
+    #-----------------
+    It FindScriptByNameWithAllowPrereleaseDownlevel {
+        $script = {
+            Find-Script -Name $script:PrereleaseTestScript -AllowPrerelease -Repository Local
```

Comment:
That's the idea.  Any use of Prerelease in any flag or version field should throw an error on downlevel.  At least, that's what the design spec said.  Do you propose a different behavior?

---
In reply to: [132066069](https://github.com/rebro-msft/PowerShellGet/pull/1#discussion_r132066069) [](ancestors = 132066069)